tmp.go

```diff
@@ -0,0 +1,6 @@
+package moira
+
+type tmp struct {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`tmp` is unused (deadcode)